.travis.yml

```diff
@@ -3,27 +3,21 @@ matrix:
  include:
  - python: 2.7
    env: DJANGO_VER_OR_TAG=lint
- - python: 2.7
-   env: DJANGO_VER_OR_TAG=django18
  - python: 2.7
    env: DJANGO_VER_OR_TAG=django1
- - python: 3.5
-   env: DJANGO_VER_OR_TAG=django18
  - python: 3.5
    env: DJANGO_VER_OR_TAG=django1
-# - python: 3.5
-#   env: DJANGO_VER_OR_TAG=django21
+ - python: 3.5
+   env: DJANGO_VER_OR_TAG=django21
  - python: 3.6
    env: DJANGO_VER_OR_TAG=lint
- - python: 3.6
-   env: DJANGO_VER_OR_TAG=django18
  - python: 3.6
    env: DJANGO_VER_OR_TAG=django1
 # django 1_11 with old MIDDLEWARE_CLASSES
  - python: 3.6
    env: DJANGO_VER_OR_TAG=django1_11_mwc
```

Comment:
I think we can remove this one as well

Reply:
which one ?

Reply:
The one with: # django 1_11 with old MIDDLEWARE_CLASSES

you have to look at the source, but I think it is: DJANGO_VER_OR_TAG=django1_11_mwc

Reply:
ok, Done